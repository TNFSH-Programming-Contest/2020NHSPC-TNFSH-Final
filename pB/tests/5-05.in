4
755.753 464.664
755.753 639.22
-572.488 639.22
-572.488 464.664
945.356 641.848
