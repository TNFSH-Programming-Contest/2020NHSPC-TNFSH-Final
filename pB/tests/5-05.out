7223427.465